src/plugins/trace/dll/DataModel/ETW/QuicEtwEvent.cs

```diff
@@ -210,13 +210,13 @@ private static QuicObjectType ComputeObjectType(TraceEvent evt)
                 case QuicEventId.ConnInFlowStats:
                     return new QuicConnectionInFlowStatsEvent(timestamp, processor, processId, threadId, pointerSize, data.ReadPointer(), data.ReadULong());
                 case QuicEventId.ConnCongestion:
-                    return new QuicConnectionCongestionEvent(timestamp, processor, processId, threadId, pointerSize, data.ReadPointer());
+                    return new QuicConnectionCongestionEvent(timestamp, processor, processId, threadId, pointerSize, data.ReadPointer(), data.ReadByte());
```

Comment:
We should have this be something like `TryReadByte()` that defaults to 0 if there's no room.

Reply:
Ditto below.

Reply:
Ouch, I thought ReadX() function already handles this as we have appended new fields before. 

Maybe we should stop appending new fields without bumping up the version. It is not that easy to handle here.